platform/authorized_api_guide.md

```diff
@@ -38,22 +73,247 @@ curl 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
+            "keyword"
+         ],
+         "siteSearchCategoryParams":[
+
+         ],
+         "delay":500,
+         "excludedIps":[
+
+         ],
+         "excludedUrlParams":[
+
+         ],
+         "excludedUserAgents":[
+
+         ],
+         "gdpr":true,
+         "gdprUserModeEnabled":false,
+         "privacyCookieDomainsEnabled":false,
+         "privacyCookieExpirationPeriod":31536000,
+         "privacyCookieDomains":[
+
+         ],
+         "organization":"default",
+         "appType":"web",
+         "gdprLocationRecognition":false
+      }
+   }
+}
+```
+ 
+#### Update app
+
+Consider you added app, but afterwards you want to change its name.
+
+Request example:
+```
+PATCH /api/apps/v2/{appId}
+```
+
+```
+curl -X PATCH 'https://<domain>/api/apps/v2/b30e538d-4b05-4a75-ae25-7eb565901f38' -H "Authorization: Bearer " -H "Content-Type: application/vnd.api+json" -v --data '{
+  "data": {
+    "attributes": {
+      "name": "NewAppName"
+    },
+    "type": "ppms/app",
+    "id": "b30e538d-4b05-4a75-ae25-7eb565901f38"
+  }
+}'
+```
+
+This request changed app name from `AppName` to `NewAppName`. Notice three things:
+* `-X PATCH` before URL. It means that this request is available using `HTTP PATCH method`
+* you have to specify also `data/id` - it's a [jsonapi]((http://jsonapi.org/)) requirement
+* also `data/type` is important. For example, when you want to work with app resource, specify it's type as `ppms/app` 
+
+There is no response example. API will return `204 No Content` status code.
+
+
+#### Delete an app
+
+Sometimes, resources are not needed anymore so lets look an example on how to delete them.
+
+```
+DELETE /api/apps/v2/{appId}
+```
+
+Request example:
+```
+curl -X DELETE 'https://<domain>/api/apps/v2/b30e538d-4b05-4a75-ae25-7eb565901f38'  -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json"
+```
+
+There is no response example. API will return `204 No Content` status code.
+
+### API usage example with Postman
+
+[Postman](https://www.getpostman.com/) is a multiplatform GUI application for creating API calls.
+PPMS allows you to export swagger documentation and easily import it to Postman.
+Depending of what you want to work with, you can import given swagger docs:
+* [Apps](https://raw.githubusercontent.com/PiwikPRO/PPMS-PublicDocs/master/platform/authorized_api/apps/index.json)
+* [Users](https://raw.githubusercontent.com/PiwikPRO/PPMS-PublicDocs/master/platform/authorized_api/users/index.json)
+* [Access control](https://github.com/PiwikPRO/PPMS-PublicDocs/blob/master/platform/authorized_api/access_control/index.json)
+
+Simply click in Postman: `import -> Import From Link`. Then, all of your paths are imported!
+You have to override two things:
+* replace your domain in url
+* add token. Click on `Authorization` tab on chosen API call and then use Bearer Token type. 
+  Paste your token, and now, you can call API using `SEND` button.
+
+## FAQ
```

Comment:
👍 bit thumbsup for this.

maybe it could be broaden with `{ "error": "access token not authorized" }` (on invalid token)